FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -6,7 +6,7 @@
 #
 
 # Logger
-from Gaudi.Configuration import INFO, DEBUG  # , VERBOSE
+from Gaudi.Configuration import INFO  # , DEBUG  # , VERBOSE
```

Comment:
is this really needed? it is convenient to switch from INFO to DEBUG "seamlessly"